common/src/main/java/org/apache/uniffle/common/util/Constants.java

```diff
@@ -23,7 +23,7 @@ private Constants() {}
 
   // the value is used for client/server compatible, eg, online upgrade
   public static final String SHUFFLE_SERVER_VERSION = "ss_v4";
-  public static final String METRICS_TAG_LABEL_NAME = "label";
```

Comment:
the name of label is not proper in proemetheus.

Reply:
cc @smallzhongfeng @xianjingfeng

Reply:
If you just change the name, the impact is not bad, but I remember that GRPC or NETTY_GRPC is available. I have tested this before. 🤔️

Reply:
I tested in this morning, but I haven't seen this. Like this,
![image](https://github.com/apache/incubator-uniffle/assets/8609142/82946054-e893-447a-bb29-613a906d75fb)

Reply:
Well, it may be that a certain PR has rolled back this function. If you can ensure that the tag parameter configuration is correct and the error will still be reported, I agree with this change.

Reply:
FYI @leixm The tag function of the online cluster should be normal now, right?